docs/developer/running-saleor/task-queue.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Task Queue
+sidebar_label: Task Queue
+---
+
+Some of the Saleor’s functionality depends on a task queue system to manage asynchronous and periodic tasks. We recommend using Celery with Redis as the message broker.
+You can use [Saleor Platform](setup/docker-compose.mdx#installation) to get this setup out of the box.
+For more information, please visit [official Celery documentation](https://docs.celeryq.dev/en/stable/index.html). 
+
+
+## Periodic tasks
+
+Celery Beat is a scheduler that triggers background tasks at regular intervals. Below is a list of Saleor features fired up by celery beat with default schedules:
+
+#### Tasks run in intervals:
+- Deactivate preorder for variants; 1 hour
+- Delete empty allocations; 1 hour
+- Delete expired reservations; 1 day
+- Delete outdated event data; 1 day
+- [Expire orders](developer/checkout/order-expiration.mdx); 5 minutes
+- [Recalculate price for catalog promotions](developer/discounts/promotions.mdx#catalogue-promotions); 30 seconds
+- [Release funds for abandoned checkouts](developer/checkout/lifecycle.mdx#releasing-funds-for-abandoned-checkouts); 10 minutes
+- Remove apps marked as removed; 3 hours
+- [Update search vectors](developer/community/contributing.mdx#searching); 20 seconds
+
+#### Tasks run at specific time:
+- Deactivate expired gift cards; at 0:00 AM
+- [Delete expired checkouts](developer/checkout/lifecycle.mdx#checkout-expiration-and-deletion); at 0:00 AM
+- [Delete expired orders](developer/checkout/order-expiration.mdx); at 2:00 AM base on 
+- Delete old exports files; once per day at 1:00 AM
+- Update stocks quantity allocated; once per day at 0:00 AM
+- [Promotion toggle](developer/discounts/promotions.mdx); based on promotion's start date and end date
+
+:::important
+It is important to understand that without proper task scheduler setup, the actions above will not be triggered.
```

Comment:
I'm wondering if we should explain what `proper task scheduler setup` means

Reply:
**Totally agree**. As we have a new doc file which explains the background tasks, we should also have the info how to setup the celery. 
Here is the task, with proposed commands to call the celery: https://github.com/saleor/saleor-docs/issues/1364. I think we could place it here 🤔